examples/annotation_import/tiled.ipynb

```diff
@@ -38,34 +38,52 @@
       },
```

Comment:
<ol><li>One thing that is lost in these updates is the tools for geospatial transformations. Here you have hard coded all of the object coordinates. But almost every ML engineer has to either start with a model prediction and project into wgs84 coordinates or start with labels and project into pixels. imo these tools are super cool and there is no great open source alternative. Even if the main point of the notebook isn't to do the projection we should highlight it. </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I see this will require a bit more work. Can I suggest demonstrating the geospatial transformations on a different notebook ? </p>

Reply:
@msokoloff1 do you think creating a new notebook that only focuses on geospatial transformations could work? Otherwise, the current notebook would be too complex, in my opinion.

Reply:
<p>Done. ! Included a polygon transformation on Step 1 below. </p>

Reply:
<p>Moved this to step 5 </p>